### PR TITLE
Replace PriceRepository with PriceIndexer

### PR DESCRIPTION
**Pull request by maciekop-l2b**
Resolves L2B-5259

### CODE REVIEW

File: packages/backend/src/modules/tracked-txs/modules/l2-costs/indexers/L2CostsPricesIndexer.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+export class L2CostsPricesIndexer extends ManagedChildIndexer {
+  constructor(private readonly $: L2CostsPricesIndexerDeps) {
+    super({ ...$, name: 'l2_costs_prices' })
+  }
+
+  override async update(from: number, to: number): Promise<number> {
+    const unixFrom = new UnixTime(from)
+    const unixTo = new UnixTime(to)
+
+    const maxRange = unixFrom.add(
```
Comment (michalsidzej) [inline packages/backend/src/modules/tracked-txs/modules/l2-costs/indexers/L2CostsPricesIndexer.ts:31]: The `CoingeckoQueryService` has a method `getAdjustedTo`. It's probably more convenient to use it here.

**Review by michalsidzej — CHANGES_REQUESTED** [review]
Simple solution, but I have a couple of nitpicks.

File: packages/backend/src/modules/tracked-txs/modules/l2-costs/repositories/L2CostsPricesRepository.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    this.autoWrap<CheckConvention<L2CostsPricesRepository>>(this)
+  }
+
+  async getAll(): Promise<L2CostsPricesRecord[]> {
+    const knex = await this.knex()
+    const rows = await knex(this.TABLE_NAME)
+    return rows.map(toRecord)
+  }
+
+  async findByTimestampRange(
```
[the remark this answers is not in the document]
Comment (maciekop-l2b) [reply packages/backend/src/modules/tracked-txs/modules/l2-costs/repositories/L2CostsPricesRepository.ts:30]: Fixed

File: packages/backend/src/peripherals/database/migrations/110_prices_for_l2_costs.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+export async function up(knex: Knex) {
+  await knex.schema.createTable('l2_costs_prices', function (table) {
+    table.dateTime('timestamp').notNullable().primary()
```
[the remark this answers is not in the document]
Comment (maciekop-l2b) [reply packages/backend/src/peripherals/database/migrations/110_prices_for_l2_costs.ts:18]: Fixed

File: packages/backend/src/modules/tracked-txs/modules/l2-costs/indexers/L2CostsPricesIndexer.ts (hunk 1 above)
Comment (maciekop-l2b) [reply packages/backend/src/modules/tracked-txs/modules/l2-costs/indexers/L2CostsPricesIndexer.ts:31]: Ahhh didn't know that. I was following PriceIndexer from TVL2

**Review by michalsidzej — APPROVED** [review]
LGTM

**Pull request merged**